### PR TITLE
lsm: Mention both `upgrade-grub2` and `grub-mkconfig` commands

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
Configuration commands for GRUB might differ depending on GRUB version
or Linux distribution. Mention both of them in the documentation.

Note: Tested on `Linux ubuntu 5.15.0-53-generic #59-Ubuntu SMP Mon Oct 17 18:53:30 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`

### CODE REVIEW

File: docs/book/programs/lsm.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ GRUB_CMDLINE_LINUX="lsm=[YOUR CURRENTLY ENABLED LSMs],bpf"
 Then rebuilding the grub configuration with:
 
 ```console
-grub-mkconfig -o /boot/grub/grub.cfg
+sudo update-grub2
```
Comment (vadorovsky) [inline docs/book/programs/lsm.md:90]: I don't see this command on openSUSE and Arch. I think we should mention both commands and ask people to double check which one is available for them.

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: docs/book/programs/lsm.md (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply docs/book/programs/lsm.md:91]: Thanks for reviewing and As per your suggestion. I have modified the document.

File: docs/book/programs/lsm.md [hunk 2]
```diff
@@ -84,8 +84,12 @@ the kernel parameters:
 GRUB_CMDLINE_LINUX="lsm=[YOUR CURRENTLY ENABLED LSMs],bpf"
 ```
 
-Then rebuilding the grub configuration with:
+Then rebuilding the grub configuration with:\
+Use any of the commands listed below depending on your system to rebuild the grub.
```
Comment (vadorovsky) [inline docs/book/programs/lsm.md:88]: This doesn't look correct. Please merge these sentences into one sentence, which fits into the rest of the doc, for example:

```suggestion
Then rebuilding the grub configuration with any of the commands listed below (each of them might be available or not in different Linux distributions):
```

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: docs/book/programs/lsm.md (hunk 2 above)
Comment (yasin-cs-ko-ak) [reply docs/book/programs/lsm.md]: @vadorovsky Thank You for your suggestions. I updated the doc.

**Comment by yasin-cs-ko-ak** [comment]
@vadorovsky Is there any other any other comments or feedback you have or is it good to merge?

**Review by vadorovsky — APPROVED** [review]
@yasin-cs-ko-ak I squashed your commits, corrected your commit and PR messages. Now it looks good, thanks!

I also added one change on top - since the most of the shell snippets in this doc use `$` and `#` to distinguish between root and non-root, I changed your `sudo` to `#` as well to keep it consistent.

https://deploy-preview-80--aya-rs.netlify.app/book/programs/lsm/

@alessandrod PTAL

**Pull request merged**